src/plugins/yajl/yajl_parse.c

```diff
@@ -269,6 +269,31 @@ static void elektraYajlParseSuppressEmpty (KeySet * returned, Key * parentKey)
 		}
 		keyDel (lookupKey);
 	}
+
+	ksRewind (returned);
+	Key * cur;
+	while ((cur = ksNext (returned)) != NULL)
+	{
+		// Not sure atm if checking for dirdata is needed or the responsibility of yajl
```

Comment:
@sanssecours can you help here?

Reply:
Please **ignore the Directory Value plugin**. If it makes your life easier Philipp, then just remove `directoryvalue` from the `infos/needs` line in the plugin contract and re-add it after you fixed issue #2132.